### DISCUSSION TITLE
PushNotificationManager.Default.CreateChannelAsync  throw COM error comment

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/WindowsAppSDK#3483] [Q&A]
it`s a very simply code follow the doc:
https://github.com/microsoft/WindowsAppSDK-Samples/tree/main/Samples/Notifications/Push
https://learn.microsoft.com/zh-cn/windows/apps/windows-app-sdk/notifications/push-notifications/push-quickstart

but
PushNotificationManager.Default.PushReceived += Default_PushReceived throw Exception
![image](https://user-images.githubusercontent.com/15758574/222903404-31f54468-0967-4561-b393-7c8cb24ef2cf.png)

PushNotificationManager.Default.Register()
![image](https://user-images.githubusercontent.com/15758574/222903452-f6aa4e8c-67c8-40e2-b542-2017cb19ff16.png)

so How Can I use  WNS base on PushNotificationManager?

**Comment by NyaMisty** [comment]
Same here
